Nimb3s.Automaton.DataAccess/Nimb3s.Automaton.DataAccess.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.78" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\Nimb3s.Automaton.Pocos\Nimb3s.Automaton.Pocos.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
after the code is moved to the [JobStatusRepository](https://github.com/nimb3s/automaton/blob/main/src/Nimb3s.Automaton.Core.Repositories.Sql/JobStatusRepository.cs) we can remove this library from the solution and from disk. all db logic will go into the [`Nimb3s.Automaton.Core.Repositories.Sql`](https://github.com/nimb3s/automaton/tree/main/src/Nimb3s.Automaton.Core.Repositories.Sql) library